documentation/reference/tensor.html

```diff
@@ -102,6 +106,18 @@ <h3 id="literal-form-examples">Literal form examples</h3>
 <pre>
 tensor(x[3]):{ {x:0}:3.0, {x:1}:5.0, {x:2}:7.0 }
 </pre>
+The same tensor in dense form:
+<pre>
+tensor(x[3]):[3.0, 5.0, 7.0]
+</pre>
+A matrix in dense form. Note that the values for the right-most dimension (y, the "column") are adjacent, so e.g
+the value 3 is here assigned to the cell {x:0,y:2}:
+<pre>
+tensor(x[2],y[3]):[[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]
+</pre>
+Also note that left and right dimension here is by alphabetical order of the dimension names, the order in which
+they are given in the type specification is irrelevant. The inner ("column") brackets are syntactic sugar and can
+be omitted.
```

Comment:
OK - but a little hard to understand - you are talking about the x and y here right? Example helps